featurebyte/feast/service/feature_store.py

```diff
@@ -50,44 +78,15 @@ def __init__(
         self.catalog_service = catalog_service
         self.online_store_service = online_store_service
 
-    async def get_feast_feature_store(
-        self,
+    @staticmethod
+    @cached(cache=feast_feature_store_cache, key=_get_feast_feature_store_cache_key)
+    async def _get_feast_feature_store(
         feast_registry: FeastRegistryModel,
-        online_store_id: Optional[ObjectId] = None,
+        feature_store: FeatureStoreModel,
+        credentials: Any,
+        online_store: Optional[FeastConfigBaseModel],
+        effective_online_store_id: ObjectId,
```

Comment:
Can `effective_online_store_id` be obtained as `online_store.id` so it doesn't have to be passed in?

Reply:
The `online_store` here actually is online store config. It does not contains ID. Will update the function to make it clearer.